components/network/quagga/quagga.p5m

```diff
@@ -0,0 +1,132 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2020 Olaf Bohlen <olbohlen@eenfach.de>
+#
+
+<transform file path=usr.*/man/.+ -> default mangler.man.stability volatile>
+set name=pkg.fmri \
+    value=pkg:/system/network/routing/quagga@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="Quagga - Quagga Routing Software"
+set name=pkg.description \
+    value="Quagga is a routing software suite, providing implementations of OSPFv2, OSPFv3, RIP v1 and v2, RIPng, ISIS and BGP-4"
+set name=com.oracle.info.description value="the Quagga routing software"
+set name=com.oracle.info.tpno value=$(TPNO)
+set name=info.classification \
+    value="org.opensolaris.category.2008:System/Administration and Configuration"
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+#set name=org.opensolaris.arc-caseid value=PSARC/2005/571
+#set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+file OpenIndiana/README.OpenIndiana path=etc/quagga/README.OpenIndiana
```

Comment:
No need to rename Solaris directory to OpenInidana, it's not a common practice. Either preserve as is or rename to files directory.

Reply:
well, since it just specifies the source of the files it does not matter anyway, so we could just keep it that way?

Reply:
It matters for maintenance, because that's what we are used to.